project1/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#Wordpress docker image with Php 7.4 and Apache
+FROM wordpress:php7.4-apache
+LABEL Maintainer = "Marcio Faria <marciodefaria@gmail.com>" \
+      Description = "LAP stack to run WordPress."
+
+# wp_data volume
+VOLUME /var/www/html
+WORKDIR /var/www/html
+
```

Comment:
I say let's test these environment variables with our DB Instance
he following environment variables are also honored for configuring your WordPress instance:

-e WORDPRESS_DB_HOST=...
-e WORDPRESS_DB_USER=...
-e WORDPRESS_DB_PASSWORD=...
-e WORDPRESS_DB_NAME=...
-e WORDPRESS_TABLE_PREFIX=...

https://hub.docker.com/_/wordpress/

Reply:
Done